apps/api/src/app/events/e2e/send-message-email.e2e.ts

```diff
@@ -1,6 +1,23 @@
 import { expect } from 'chai';
-import { CompileEmailTemplate } from '../../content-templates/usecases/compile-email-template/compile-email-template.usecase';
-import { createMailData } from '../usecases/send-message/send-message-email.usecase';
+import { CompileEmailTemplate } from '@novu/application-generic';
+import { IEmailOptions } from '@novu/stateless';
+
+export const createMailData = (options: IEmailOptions, overrides: Record<string, any>): IEmailOptions => {
+  const filterDuplicate = (prev: string[], current: string) => (prev.includes(current) ? prev : [...prev, current]);
+
+  let to = Array.isArray(options.to) ? options.to : [options.to];
+  to = [...to, ...(overrides?.to || [])];
+  to = to.reduce(filterDuplicate, []);
+
+  return {
+    ...options,
+    to,
+    from: overrides?.from || options.from,
+    text: overrides?.text,
+    cc: overrides?.cc || [],
+    bcc: overrides?.bcc || [],
+  };
+};
```

Comment:
send-message-email is only used in the `worker-service` and here in the API test we can import that dependency, that's why this code is copied